src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -96,5 +89,32 @@
             </ul>
         </div>
     </div>
-    <tif:feedbackSessionPublishButton publishButton="${actions.publishButton}" buttonType="btn-default btn-xs margin-bottom-7px" />
-</c:if>
+    <div title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTS %>"
+         data-toggle="tooltip"
+         data-placement="top"
+         style="display: inline-block; padding-right: 5px;">
+        <div class="btn-group margin-bottom-7px">
+            <a class="btn btn-default btn-xs btn-tm-actions session-view-for-test"
+               href="${actions.resultsLink}">
+                Results
+            </a>
+            <button type="button"
+                    class="btn btn-default btn-xs btn-tm-actions dropdown-toggle"
+                    data-toggle="dropdown"
+                    aria-expanded="false">
+                <span class="caret"></span>
+            </button>
+            <ul class="dropdown-menu" role="menu">
+                <li>
+                    <a class="session-view-for-test"
+                     href="${actions.resultsLink}">
+                      View Results
+                  </a>
```

Comment:
The indentation for `</a>` seems off. Could maybe combine lines 109-112 into a single line.